frontend/src/app/search/[cid]/page.tsx

```diff
@@ -0,0 +1,39 @@
+'use client'
+
+import { useEffect, useState } from "react";
+import { ApiService } from "../../../services/api";
+import FileCard from "../../../components/common/FileCard";
+import { UploadedObjectMetadata } from "../../../models/UploadedObjectMetadata";
+
+
+export default function Page({ params: { cid } }: { params: { cid: string } }) {
+
+    const [objectsMetadata, setObjectsMetadata] = useState<UploadedObjectMetadata[]>();
+
+    useEffect(() => {
+        ApiService.searchHeadCID(cid).then((e) => Promise.all(e.map(e => ApiService.fetchUploadedObjectMetadata(e)))).then(setObjectsMetadata);
+    }, []);
+
+
```

Comment:
Can you add prettier and editor.formatOnSave setting?

Reply:
On a new PR